### PR TITLE
VuePress transition

### PR DESCRIPTION
**Pull request by khronokernel**

### CODE REVIEW

**Comment by khronokernel** [comment]
To build the site:

```
brew install yarn
yarn add vuepress -D
yarn install
yarn dev
```

File: .travis.yml [hunk 1]
```diff
@@ -3,13 +3,11 @@ node_js: lts/*
 cache:
```
Comment (dhinakg) [inline .travis.yml:3]: TODO: Properly merge Travis setup from laptop guide in

File: .vuepress/components/Foo/Bar.vue [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+<script>
+  export default {
+    data () {
+      return {
+        msg: 'Hello this is <Foo-Bar>'
+      }
+    }
+  }
+</script>
```
Comment (dhinakg) [inline .vuepress/components/Foo/Bar.vue:15]:
```suggestion
```
Useless

File: .vuepress/components/OtherComponent.vue [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<template>
+  <p class="demo">This is another component</p>
+</template>
```
Comment (dhinakg) [inline .vuepress/components/OtherComponent.vue:3]:
```suggestion
```
Useless

File: .vuepress/components/demo-component.vue [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+<script>
+  export default {
+    data() {
+      return {
+        msg: 'Hello this is <demo-component>'
+      }
+    }
+  }
+</script>
```
Comment (dhinakg) [inline .vuepress/components/demo-component.vue:15]:
```suggestion
```
Useless

File: .vuepress/config.js [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+    theme: 'vuepress-theme-succinct',
+    globalUIComponents: [
+        'ThemeManager'
+    ],
+
+    themeConfig: {
+        lastUpdated: true,
+        repo: 'https://github.com/dortania/OpenCore-Install-Guide',
+        editLinks: false,
+        docsDir: '',
```
Comment (dhinakg) [inline .vuepress/config.js:63]: Need to look at docs and see if this needs to be changed to anything

File: .vuepress/enhanceApp.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+ */
+
+export default ({
+  Vue, // the version of Vue being used in the VuePress app
+  options, // the options for the root Vue instance
+  router, // the router instance for the app
+  siteData // site metadata
+}) => {
+  // ...apply enhancements for the site.
+}
```
Comment (dhinakg) [inline .vuepress/enhanceApp.js:14]:
```suggestion
```
Useless

File: .vuepress/styles/index.styl [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  max-width 1000px
+
+.navbar
+.links
+  background-color var(--navBgColorDark)!important
+
+.navbar
+  filter: drop-shadow(0px 1px 1px var(--bgColor));
+
+*/
```
Comment (dhinakg) [inline .vuepress/styles/index.styl:38]:
```suggestion
```
Remove comments?

File: .vuepress/styles/index.styl [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+  pre, pre[class*="language-"]
+    margin-top 0
+
+html {
+  overflow:   scroll;
+}
+::-webkit-scrollbar {
+    width: 0px;
+    background: transparent; /* make scrollbar transparent */
+}
```
Comment (dhinakg) [inline .vuepress/styles/index.styl:56]:
```suggestion
html
  overflow scroll

::-webkit-scrollbar
    width 0px
    background transparent /* make scrollbar transparent */
```

File: .vuepress/styles/index.styl [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+.links
+  background-color var(--navBgColorDark)!important
+
+.navbar
+  filter: drop-shadow(0px 1px 1px var(--bgColor));
+
+*/
+  
+.navbar
+  filter: drop-shadow(0px 1px 1px var(--bgColor));
```
Comment (dhinakg) [inline .vuepress/styles/index.styl:41]:
```suggestion
  filter drop-shadow(0px 1px 1px var(--bgColor))
```

File: .vuepress/styles/index.styl [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+   &:nth-child(2n)
+     background-color var(--sideBgColor)
+
+	 
+
+.dropdown-wrapper .nav-dropdown
+  border: 1px solid var(--sideBgColor)!important
+  background-color var(--sideBgColor)!important
+  box-shadow: 0px 0px 2px var(--bodyBgColor)
+  right: -12%!important
```
Comment (dhinakg) [inline .vuepress/styles/index.styl:70]:
```suggestion
  border 1px solid var(--sideBgColor)!important
  background-color var(--sideBgColor)!important
  box-shadow 0px 0px 2px var(--bodyBgColor)
  right -12%!important
```
Is `!important` needed here?

File: .vuepress/styles/palette.styl [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+//$borderColor = #eaecef
+$codeBgColor = #282c34
+
+$bgColor = #2d3033
+$navBgColor = #363b40
+
+$bgColorDark = #2d3033
+$navBgColorDark = #363b40
+$sideBgColorDark = #363b40
+*/
```
Comment (dhinakg) [inline .vuepress/styles/palette.styl:38]:
```suggestion
```
Remove comments?

File: .vuepress/styles/palette.styl [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+$bgColor = #111112
```
Comment (dhinakg) [inline .vuepress/styles/palette.styl:40]: Possibly rename this? Kinda ambiguous as current

File: config.plist/README.md [hunk 13]
```diff
@@ -54,62 +54,53 @@ Now comes the important part, selecting the configuration path. Each platform ha
...
 
-[Zen](/AMD/zen.md)
-
-* 1XXX, 2XXX, 3XXX series, 2017-2020 era
 * Note: Threadripper 3rd gen(39XX) are not supported, 1st and 2nd gen however are supported
 
-### Misc
+### Legacy
+
+Note that this is only guide-lines and will not be full-features guides like the other generations. This is mainly due to not enough testing, verifiable information and overall community support as these hardware are become less and less used.
```
Comment (dhinakg) [inline config.plist/README.md:99]:
```suggestion
Note that these are only guidelines and will not be full-featured guides like the other generations. This is mainly due to not enough testing, verifiable information, and overall community support as this hardware become less and less used.
```

File: package.json [hunk 14]
```diff
@@ -1,46 +1,29 @@
 {
```
Comment (dhinakg) [inline package.json:1]: Can we leave the name/version/description/etc? No need to delete them

File: package.json [hunk 15]
```diff
@@ -1,46 +1,29 @@
...
   "name": "OpenCore-Desktop-Guide",
```
Comment (dhinakg) [inline package.json:2]:
```suggestion
  "name": "OpenCore-Install-Guide",
```

File: package.json [hunk 16]
```diff
@@ -1,46 +1,29 @@
...
-    "run-script-os": "^1.1.1",
-    "spellchecker-cli": "^4.2.0"
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "authors": {
+    "name": "",
+    "email": ""
   },
+  "repository": "/OpenCore-Desktop-Guide",
```
Comment (dhinakg) [inline package.json:10]:
```suggestion
  "repository": "/OpenCore-Install-Guide",
```

File: package.json [hunk 17]
```diff
@@ -1,46 +1,29 @@
...
-    "type": "git",
-    "url": "git+https://github.com/dortania/OpenCore-Desktop-Guide.git"
+    "dev": "vuepress dev",
+    "build": "vuepress build"
   },
-  "author": "Dortania",
-  "license": "UNLICENSED",
-  "bugs": {
-    "url": "https://github.com/dortania/OpenCore-Desktop-Guide/issues"
+  "license": "MIT",
```
Comment (dhinakg) [inline package.json:15]:
```suggestion
  "license": "UNLICENSED",
```
Need to import the license from laptop guide

File: package.json (hunk 14 above)
Comment (dhinakg) [reply package.json:1]: Wdym resolved

File: extras/msr-lock.md [hunk 18]
```diff
@@ -12,16 +12,14 @@ CFG-Lock is a setting in your BIOS that allows for a specific register(in this c
 
 So to fix it we have 2 options:
 
-1. Patch macOS to work with our hardware
-
+* Patch macOS to work with our hardware
 * This creates instability and unnecessary patching for many
 * The 2 patches we use for this:
   * `AppleCpuPmCfgLock` for AppleIntelPowerManagement.kext
   * `AppleXcpmCfgLock` for the Kernel(XNU)
```
Comment (dhinakg) [inline extras/msr-lock.md:20]:
```suggestion
	* This creates instability and unnecessary patching for many
	* The 2 patches we use for this:
	  * `AppleCpuPmCfgLock` for AppleIntelPowerManagement.kext
	  * `AppleXcpmCfgLock` for the Kernel(XNU)

```

**Pull request merged**